scst-store/configuration/use-node-port.hbs.md

```diff
@@ -0,0 +1,23 @@
+# Using NodePort
+
+**Note:** NodePort is only recommended when: the cluster does not support ingress or the cluster does not support `LoadBalancer` type to services.  `NodePort` is not supported for a multi-cluster setup, as certificates cannot be modified.
```

Comment:
Same comment as above.

As part of the introduction, you should mention that the recommended service type is ingress and link to the new `ingress.hbs.md`.